src/network/connectionpool.py

```diff
@@ -116,6 +128,7 @@ def startListening(self, bind=None):
         self.listeningSockets[ls.destination] = ls
 
     def startUDPSocket(self, bind=None):
+        """"""
```

Comment:
```suggestion
        """Open an UDP socket. Depending on settings, it can either only accept incoming UDP packets, or also be able to send them."""
```